helm/prometheus-rules/templates/recording-rules/gs-managed-app-deployment-status.rules.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: gs-managed-app-deployment-status.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: gs-managed-app-deployments
+    rules:
+    - expr: kube_deployment_status_replicas_available *
+            on (deployment) group_left(label_giantswarm_io_service_type, label_app)
```

Comment:
Can you not use the app label instead of label_app ? This should not be

Reply:
`label_app` is a is a label (or field?) of the `kube_deployment_labels` metric which reflects the actual kubernetes label `app` of the deployment on the cluster.

Should we maybe rename the `label_app` to something else?

Reply:
If possible, I would because this is confusing with the app label on the metric. @giantswarm/team-atlas what do you think here ? Would it make sense to have a label called managed_app=app_name ?

Reply:
yeah makes sense. Used `label_replace` to rename to `managed_app`